doc/classes/EditorSettings.xml

```diff
@@ -1063,10 +1063,16 @@
 		<member name="text_editor/behavior/navigation/stay_in_script_editor_on_node_selected" type="bool" setter="" getter="">
 			If [code]true[/code], prevents automatically switching between the Script and 2D/3D screens when selecting a node in the Scene tree dock.
 		</member>
+		<member name="text_editor/behavior/navigation/use_custom_separators" type="bool" setter="" getter="">
+			If [code]true[/code], using [kbd]Ctrl + Left[/kbd] or [kbd]Ctrl + Right[/kbd] ([kbd]Cmd + Left[/kbd] or [kbd]Cmd + Right[/kbd] on macOS) bindings will also stop moving caret if a character within [member text_editor/behavior/navigation/word_separators] is detected. Useful for subword moving.
```

Comment:
Maybe document the behavior when this option is false? You say "if true, it will *also* stop ..." but it's not clear what this "also" refers to, nor what actually happens on "false".

Reply:
updated